app/src/main/java/de/westnordost/streetcomplete/quests/construction/MarkCompletedBuildingConstruction.kt

```diff
@@ -33,7 +33,7 @@ class MarkCompletedBuildingConstruction : OsmFilterQuestType<CompletedConstructi
                     changes.modify("building", value)
                     deleteTagsDescribingConstruction(changes)
                 } else {
-                    changes.addOrModify(SURVEY_MARK_KEY, LocalDate.now().toCheckDateString())
+                    changes.updateCheckDate()
```

Comment:
This will remove other non-standard check dates, which I assume is a good thing?